corehq/apps/cleanup/management/commands/evaluate_couch_model_for_sql.py

```diff
@@ -34,24 +28,36 @@ def add_arguments(self, parser):
         )
 
     def handle(self, doc_type, **options):
-        attrs = options.get('attrs', [])
         db = couch_config.get_db(options.get('db', None))
-        blank_counts = defaultdict(lambda: 0)
+        key_counts = defaultdict(lambda: 0)
         max_lengths = defaultdict(lambda: 0)
 
         print("Found {} {} docs\n".format(get_doc_count_by_type(db, doc_type), doc_type))
 
+        def _evaluate_doc(doc, prefix=None):
+            for key, value in doc.items():
+                if prefix:
+                    key = f"{prefix}.{key}"
+                if not value or isinstance(value, bool):
```

Comment:
The documentation for this command says "whether those attributes are ever null", but this will match many things that are not null. Would it make sense to update the wording in the docstring to be "whether those attributes are ever _null, empty or boolean_"? (emphasis added)